Electron-Basic-Video-Chat/openTok.js

```diff
@@ -0,0 +1,39 @@
+//Set Credentials
+const apiKey = "";
+const sessionId = "";
+const token = "";
+
+//Initialize Session
+const session = OT.initSession(apiKey, sessionId);
+
+//Set session event listeners
+session.on({
+  streamCreated: (event) => {
+    session.subscribe(event.stream, 'subscriber', (error) => {
+      if (error) console.log(`There was an issue subscribing to the stream ${event}`);
+    });
+  },
+  streamDestroyed: (event) => {
+    console.log(`Stream ${event.stream.name} ended ${event.reason}`);
```

Comment:
I think some better sentence structure could go a long way for the console logs when you're interpolating event params in this sample repo. Otherwise the intent of the injection is not clear.

e.g. `Stream with name: ${name} ended because of reason: ${reason}`

Reply:
fixed in the latest commit